.circleci/config.yml

```diff
@@ -1,4 +1,14 @@
+version: 2
+
 aliases:
+  - &update-go-to-1-12
```

Comment:
we use the official circle ci go images (faster that way) , circleci/golang:1.12.6 

any reason for doing this?

Reply:
You have more RAM with the machine executor and it does not allow use of `circleci/golang:1.12.6` :/.

More RAM allows us use of:
1. Parallezation (see changed `lint-go` task) otherwise it will fail with "not enough memory" error
1. Execution tests in bulk (currently we searching pkgs and run tests on them separately) 

These things should speed up the build significantly (hopefully).

You also can use docker-compose with machine exec, in turn, that would allow us to use devenv blocks inside the CI. Currently we recreate these environments in circle config by using some parts of the devenv configuration and their sql dumps.

Docker-compose bit I was planning to discuss during the weekly backend meeting by using this pr as demonstrative example

Reply:
ah, interesting.